Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Header.jinja

```diff
@@ -443,35 +443,31 @@ namespace {{ Component.attrib['Namespace'] }}
         {{ DeclareNetworkPropertyAccessors(Component, 'Autonomous', 'Authority', false)|indent(8) -}}
         {{ DeclareNetworkPropertyAccessors(Component, 'Autonomous', 'Authority', true)|indent(8) -}}
         {{ DeclareArchetypePropertyGetters(Component)|indent(8) -}}
-        {{ DeclareRpcInvocations(Component, 'Client', 'Authority', false)|indent(8) -}}
-        {{ DeclareRpcInvocations(Component, 'Client', 'Authority', true)|indent(8) -}}
         {{ DeclareRpcInvocations(Component, 'Autonomous', 'Authority', false)|indent(8) -}}
         {{ DeclareRpcInvocations(Component, 'Autonomous', 'Authority', true)|indent(8) -}}
         {{ DeclareRpcInvocations(Component, 'Authority', 'Autonomous', false)|indent(8) -}}
         {{ DeclareRpcInvocations(Component, 'Authority', 'Autonomous', true)|indent(8) -}}
         {{ DeclareRpcInvocations(Component, 'Authority', 'Client', false)|indent(8) -}}
         {{ DeclareRpcInvocations(Component, 'Authority', 'Client', true)|indent(8) -}}
+
+        //! RPC Handlers: Override handlers in order to implement what happens after receiving an RPC
         {{ AutoComponentMacros.DeclareRpcHandlers(Component, 'Server', 'Authority', false)|indent(8) -}}
-        {{ AutoComponentMacros.DeclareRpcHandlers(Component, 'Client', 'Authority', false)|indent(8) -}}
         {{ AutoComponentMacros.DeclareRpcHandlers(Component, 'Autonomous', 'Authority', false)|indent(8) -}}
         {{ AutoComponentMacros.DeclareRpcHandlers(Component, 'Authority', 'Autonomous', false)|indent(8) -}}
-        {{ AutoComponentMacros.DeclareRpcSignals(Component, 'Server', 'Authority')|indent(8) -}}
-        {{ AutoComponentMacros.DeclareRpcSignals(Component, 'Client', 'Authority')|indent(8) -}}
-        {{ AutoComponentMacros.DeclareRpcSignals(Component, 'Autonomous', 'Authority')|indent(8) -}}
```

Comment:
We really need to document the supported directions for both network properties and rpc, so that this doesn't stay a secret knowledge.

Reply:
For network properties it should be simple. Authority to everything:
'Authority' to 'Server' (migration properties)
'Authority' to 'Autonomous' (autonomous only properties)
'Authority' to 'Client', and (client 'simulated' proxy properties)
'Autonomous' to 'Authority' (replicating stuff like metrics from the autonomous up to the authority)

For RPC's it's a bit less structured: ('Authority' to 'Server' is left out for lack of a good use-case at this time)
'Authority' to 'Autonomous' (corrections, etc..)
'Authority' to 'Client' (effects, explosions and such..)
'Server' to 'Authority' (inter-entity communication, DealDamage, ApplyImpulse, etc..)
'Client' to 'Authority' (? I think we support this but I don't have a good use-case off the top of my head)
'Autonomous' to 'Authority' (Inputs and stuff)

Reply:
Great idea! I've created an internal documentation task so this isn't lost. LYN-8163

Reply:
Too be clear, the change removes client > authority. Like you stated here there wasn’t a use case and we can always add it back if someone presents a need.

Reply:
I'm for removing client > auth for the time being